src/components/tree/phyloTree/phyloTree.ts

```diff
@@ -8,9 +8,89 @@ import * as grid from "./grid";
 import * as confidence from "./confidence";
 import * as labels from "./labels";
 import * as regression from "./regression";
+import { Callbacks, Layout, Params, PhyloNode, ReduxNode, ScatterVariables, Visibility } from "./types";
+import { Selection } from "d3";
+
+export interface PhyloTree {
+  grid: boolean
+  attributes: string[]
+  params: Params
+  groups: {
+    branchGradientDefs?: Selection<SVGDefsElement, unknown, null, any>
+    branchStem?: Selection<SVGDefsElement, unknown, null, any>
+    branchTee?: Selection<SVGDefsElement, unknown, null, any>
+    clipPath?: Selection<SVGDefsElement, unknown, null, any>
+    confidenceIntervals?: Selection<SVGDefsElement, unknown, null, any>
+    regression?: Selection<SVGDefsElement, unknown, null, any>
+    tips?: Selection<SVGDefsElement, unknown, null, any>
+    vaccines?: Selection<SVGDefsElement, unknown, null, any>
+  }
+  id: string
+  nodes: PhyloNode[]
+  zoomNode: PhyloNode
+  dateRange: [number, number]
+  strainToNode: Record<string, PhyloNode>
+  change: typeof change
+  modifySVG: typeof modifySVG
+  modifySVGInStages: typeof modifySVGInStages
+  render: typeof renderers.render
+  clearSVG: typeof renderers.clearSVG
+  setClipMask: typeof renderers.setClipMask
+  drawTips: typeof renderers.drawTips
+  drawBranches: typeof renderers.drawBranches
+  drawVaccines: typeof renderers.drawVaccines
+  drawRegression: typeof renderers.drawRegression
+  removeRegression: typeof renderers.removeRegression
+  updateColorBy: typeof renderers.updateColorBy
+  setDistance: typeof layouts.setDistance
+  setLayout: typeof layouts.setLayout
+  rectangularLayout: typeof layouts.rectangularLayout
+  scatterplotLayout: typeof layouts.scatterplotLayout
+  unrootedLayout: typeof layouts.unrootedLayout
+  radialLayout: typeof layouts.radialLayout
+  setScales: typeof layouts.setScales
+  mapToScreen: typeof layouts.mapToScreen
+  calculateRegression: typeof regression.calculateRegression
+  removeConfidence: typeof confidence.removeConfidence
+  drawConfidence: typeof confidence.drawConfidence
+  drawSingleCI: typeof confidence.drawSingleCI
+  drawBranchLabels: typeof labels.drawBranchLabels
+  removeBranchLabels: typeof labels.removeBranchLabels
+  updateBranchLabels: typeof labels.updateBranchLabels
+  updateTipLabels: typeof labels.updateTipLabels
+  removeTipLabels: typeof labels.removeTipLabels
+  hideGrid: typeof grid.hideGrid
+  addGrid: typeof grid.addGrid
+  showTemporalSlice: typeof grid.showTemporalSlice
+  hideTemporalSlice: typeof grid.hideTemporalSlice
+
+  confidencesInSVG: boolean
+  regression?: regression.Regression
+  layout: Layout
+  svg: Selection<SVGSVGElement | null, unknown, null, any>
+  distance: "num_date" | "div"
```

Comment:
you define a `Distance` type in `./types`; use it here.

Reply:
Done: https://github.com/nextstrain/auspice/blob/3cf6768919650f4294a8c7a213cfa11dda5fc462/src/components/tree/phyloTree/types.ts#L268